baseapp/baseapp.go

```diff
@@ -175,12 +174,14 @@ func NewBaseApp(
 		app.SetMempool(mempool.NoOpMempool{})
 	}
 
-	if app.processProposal == nil {
-		app.SetProcessProposal(app.DefaultProcessProposal())
-	}
+	abciProposalHandler := NewDefaultProposalHandler(app.mempool, app)
 
 	if app.prepareProposal == nil {
-		app.SetPrepareProposal(app.DefaultPrepareProposal())
+		app.SetPrepareProposal(abciProposalHandler.PrepareProposalHandler())
```

Comment:
Could you update the comments in `app.go` and `app_v2.go` to match the new API: https://github.com/cosmos/cosmos-sdk/blob/a90569c7e4e4282005d3596cc2763c5f6b2e3f16/simapp/app_v2.go#L176

Reply:
And here too: https://github.com/cosmos/cosmos-sdk/blob/f211bd865237885d968a9eecd38ac80cc0849fe7/docs/docs/building-apps/02-app-mempool.md#L56-L57